packages/@aws-cdk/aws-cloudwatch/lib/private/util.ts

```diff
@@ -0,0 +1,10 @@
+export function dropUndefined<T extends object>(x: T): T {
```

Comment:
Extending this everywhere honestly sounds kind of exhausting to me, and I'm not sure I see the benefit. Are you really proposing to have a file called `object.ts` with a single function in it, and a file called `string.ts` with a single function in it, because you're opposed to having a file called `util.ts` with both?

I know that the argument against this is "wild growth" of functions and before you know it there will be 3,351 functions in there, but consider:

* Our modules are fairly small and they don't share utility functions; they're all copy/pasted around. If a module is going to have 10-odd utility functions, that's going to be at the upper end of the scale.
* Even if there will be many more, the effort of splitting up the single file into multiple ones is going to be trivial, *because* of this local use. The effect is guaranteed to not permeate into any other package.

My personal opinion: the level of scrutiny given to this issue is not proportional to its impact, nor would the proposed name be clearer to me (personally). Given just the filename `object.ts` and nothing else, I would struggle to think what could be in there. Nevertheless, I will apply the change.